lib/utils.js

```diff
@@ -87,20 +87,23 @@ var Utils = module.exports = {
     var timeZone = null;
     return SqlString.formatNamedParameters(sql, parameters, timeZone, dialect);
   },
-  cloneDeep: function(obj, fn) {
+  cloneDeep: function(obj) {
+    obj = obj || {};
     return _.cloneDeepWith(obj, function (elem) {
-      // Do not try to customize cloning of plain objects and strings
+      // Do not try to customize cloning of arrays or POJOs
       if (Array.isArray(elem) || _.isPlainObject(elem)) {
         return undefined;
       }
+
+      // Don't clone stuff that's an object, but not a plain one - fx example sequelize models and instances
+      if (typeof elem === 'object' && !_.isPlainObject(elem)) {
```

Comment:
I don't think it should be needed - If it is a POJO it will hit the first if, so the only type of object we will see here are non-plain